content/reference/dos-mitigation.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "DOS Mitigation"
+weight: 4
+---
+
+In general we have to design our protocols to be resilient to malicious or
+misbehaving nodes. libp2p offers reasonable defaults and primitives tools let the
+end application developer design resilient peer-to-peer programs.
+
+# General tactics that work across libp2p implementations
+
+## Limit the number of concurrent streams your protocol needs
```

Comment:
Optional: I think a similar section on the number of connections would be helpful.

Reply:
Agreed.